docs/cs_v6_release_guide.md

```diff
@@ -447,6 +447,36 @@ sed -i "s|detail->sysz|detail->systemz|g" $1
 
 Write it into `rename.sh` and run it on files with `sh rename.sh <src-file>`
 
+#### Python binding compatibility
+
+The Python bindings were changed to match the renamed architectures. To allow using the old `ARM64_*` and `SYSZ_*` constants in scripts running against Capstone v6+, compatibility modules can be imported, which monkey-patch the modules to export the same constants with the old name as well. The compatibility module has to be imported **before** anything else from capstone.
```

Comment:
Split up the ` To allow using the old...` sentence into multiple please. It is quite long and might be challenging for non-native speakers.

Reply:
Also the typo `capstone` -> `Capstone`